### DISCUSSION TITLE
Support channel for sponsors

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#595] [Ideas]
So,

I became a sponsor a while back to support the developmental effort.  I've got EOS on three machines at present.  It has become my distro of choice.

When "stable-channel" builds were announced via e-mail, I downloaded it and installed on an old Dell XPS 13 laptop.

Everything was great until a power related "event" has corrupted something prohibiting boot into the DE.

That's not the issue here, but if Elementary is going to have an "official" sponsor system, there should be a corresponding support channel not available to the general public.

I've posted messages about my issue with virtually no response. I posted the same message on Reddit and one poster suggested something to try. 

Frankly, we (Elementary et al) need to do better.

**Comment by danirabbit** [comment]
There is a sponsors only chat available on discord but sponsoring elementary on GitHub doesn't include paid support. There simply aren't enough funds to offer that sort of thing. We'd need at least twice our current income to provide something like that. The only thing we can offer at the moment is volunteer community support

**Comment by doobes** [comment]
Well then,

There is more than one person having boot issues with v8.0.

Might be something you want to take a gander at.....

'just sayin'

**Reply by danirabbit** [reply]
We work as quickly as possible to resolve issues with the limited resources we have. Your patience is appreciated since the vast majority of development is done on a completely voluntary basis.

Please remember that early access builds are for testing purposes as a perk of sponsorship but should not be considered production ready

**Comment by doobes** [comment]
Understand completely.

Would it be constructive to have some sort of "isolated" place to discuss issues which develop during testing, like the one that skytetsu and I have had?

I don't mind being a tester if my input is useful. 

In that frame of mind, my laptop is sitting here.  That way, I can provide more info in the diagnosis.

I'm not going to reinstall until there is a production version. While I didn't (and won't ever) lose any data, I have lost time.

cheers

chris

**Comment by vjr** [comment]
hello @doobes - a fellow elementary OS fan and user here :-) i see a few github tickets you've opened looking for help using/testing the OS - may i suggest you join the discord community (if you're not already there) and (re)post your questions there? you might get quicker advice there actually - it appears to be a growing and enthusiastic community over there. give it a try and see if it helps you make progress?

**Comment by doobes** [comment]
I posted a message yesterday. 

I'll be honest, I'm an old fart and find it difficult to wrap my head around Discord. Seems like an awful lot going on with little, to no, "organization", for lack of a better word.

**Reply by svandragt** [reply]
It's not too much different than a BBS like we had on telnet, honestly.

**Comment by vjr** [comment]
dont be discouraged by your initial impression of it :-) keep going over there, if not immediately, someone who has a suggestion might eventually respond with some advice.
